src/code/z_sram.c

```diff
@@ -308,11 +308,11 @@ static s16 sDungeonEntrances[] = {
 /**
  *  Copy save currently on the buffer to Save Context and complete various tasks to open the save.
  *  This includes:
- *  - Set proper entrance depending on where the game was saved
- *  - If health is less than 3 hearts, give 3 hearts
- *  - If either scarecrow song is set, copy them from save context to the proper location
- *  - Handle a case where the player saved and quit after zelda cutscene but didnt get the song
- *  - Give and equip master sword if player is adult and doesnt have kokiri sword (bug?)
+ *  - Set proper entrance depending on where the game was saved.
+ *  - If health is less than 3 hearts, give 3 hearts.
+ *  - If either Scarecrow's Song is set, copy it from SaveContext to the proper location.
```

Comment:
may want to reword that sentence better, it's trying to cover the two scarecrow songs

Reply:
`- Copy any set Scarecrow's Songs from SaveContext to the proper location.`?

Reply:
I like the first one